ValidateJobs.py

```diff
@@ -0,0 +1,143 @@
+import yaml
```

Comment:
Perhaps a shebang?

Reply:
so like this: `#!/usr/bin/env python3`?

Reply:
That looks good!

Reply:
Also make the file executable.

Reply:
Ok, now i'm not sure what you mean?

Reply:
`chmod +x ValidateJobs.py` will make the file executable, along with the shebang you can run the script without needing to do `python ValidateJobs.py`